File: 백준/Bronze/1159. 농구 경기/농구 경기.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin");
// let input = fs.readFileSync("./input.txt");
input = input.toString().trim().split("\n");
input.sort();

const N = parseInt(input[0]);
let cnt = {};
for (let i = 1; i <= N; i++) {
  first = input[i][0];
  if (first in cnt) {
    cnt[first] += 1;
  } else {
    cnt[first] = 1;
  }
}

answer = "";
for (let i in cnt) {
  if (cnt[i] >= 5) {
    answer += i;
  }
}

if (answer === "") {
  console.log("PREDAJA");
} else {
  console.log(answer);
}

File: 백준/Bronze/1159. 농구 경기/농구 경기.py
import sys
# sys.stdin = open("input.txt", 'r')
input = sys.stdin.readline

N = int(input())
cnt = {}
answer = []

for _ in range(N):
    S = input().rstrip()
    first = S[0]
    if first in cnt:
        cnt[first] += 1
    else:
        cnt[first] = 1

for i in cnt:
    if cnt[i] >= 5:
        answer.append(i)

answer.sort()
if answer:
    for ans in answer:
        print(ans, end="")
else:
    print("PREDAJA")